devel/hs-git-brunch/Makefile

```diff
@@ -1,15 +1,15 @@
 PORTNAME=	git-brunch
-PORTVERSION=	1.5.0.0
+PORTVERSION=	1.5.1.0
 CATEGORIES=	devel haskell
 
 MAINTAINER=	haskell@FreeBSD.org
-COMMENT=	git branch checkout command line tool
+COMMENT=	Git branch checkout command line tool
 
 LICENSE=	BSD3CLAUSE
 
 USES=		cabal
 
-USE_CABAL=	HUnit-1.6.2.0 \
+USE_CABAL= 	HUnit-1.6.2.0 \
```

Comment:
Is this space instead of a tab?

Reply:
It is... `portlint` did not saw it. See below for a commit without the space.